pollux/core/src/main/scala/org/hyperledger/identus/pollux/core/service/CredentialServiceImpl.scala

```diff
@@ -223,7 +223,7 @@ class CredentialServiceImpl(
         claims = attributes,
         thid = thid,
         UUID.randomUUID().toString,
-        "domain",
+        "domain", // TODO remove the hardcoded domain
```

Comment:
should we add the domain to the config file or make it configurable for the tenant?

Reply:
https://github.com/hyperledger/identus-cloud-agent/issues/1422